account-integrations/safe/src/SafeECDSAPlugin.sol

```diff
@@ -71,10 +72,17 @@ contract SafeECDSAPlugin is BaseAccount {
         return _owner;
     }
 
+    function updateOwner(address newOwner) public {
+        // TODO: Check if msg.sender is the Safe. We can't do that now because the Safe
+        // and the plugin are deployed at the same time.  So the plugin doesn't know the Safe address.
+        emit OWNER_UPDATED(_owner, newOwner);
+        _owner = newOwner;
+    }
```

Comment:
Lets make sure the readme is clear with its state of "pre-alpha" and "not for production use" etc.